components/automate-ui/src/app/modules/infra-proxy/org-users/org-users.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, Input, OnInit, OnDestroy, EventEmitter, Output } from '@angular/core';
+import { Subject } from 'rxjs';
+import { LayoutFacadeService, Sidebar } from 'app/entities/layout/layout.facade';
+
+@Component({
+  selector: 'app-org-users',
+  templateUrl: './org-users.component.html',
+  styleUrls: ['./org-users.component.scss']
+})
+
+export class OrgUsersComponent implements OnInit, OnDestroy {
+  @Input() serverId: string;
+  @Input() orgId: string;
+  @Output() resetKeyRedirection = new EventEmitter<boolean>();
+
+  public users;
```

Comment:
@chaitali-mane 
can you please define types here 
```
public users;
  public usersListLoading = false;
  public authFailure = false;
  public loading = false;
  public current_page = 1;
  public per_page = 100;
  public total: number;
```

Reply:
Updated variables types